### ISSUE TITLE
Remove universal=1 from wheel config.

### ISSUE THREAD

**Issue by apollo13** [timothycrosley/pies#41]
https://github.com/timothycrosley/pies/blob/develop/setup.cfg#L2 is wrong and should be false (or removed) for this package. The reason is simple: ``install_requires`` changes based on whether python2 or python3 is used to execute setup.py -- this will have the resulting wheel contain requirements which are wrong depending which python version you use to install it.

**Comment by dstufft** [comment]
You could also just use enviornment markers to have a conditional dependencies:

I think you just need to add:

```python
extras_require={
    ":python_version == 2.6 or python_version == 2.7": ["enum34"],
}
```

to your setup.py.

**Comment by timothycrosley** [comment]
@apollo13 thanks for reporting this issue! And @dstufft thanks for your insight on the best way to resolve it. I've just released version 2.6.5 which should use this approach to correctly build the universal wheel.

Thanks!

~Timothy

**Comment by apollo13** [comment]
@timothycrosley The wheel still has:
```
Requires: pies2overrides
Requires: enum34
Requires-Dist: pies2overrides
Requires-Dist: enum34
```
I think you can should drop those two from `install_requires` completely.
